sql-files/main.sql

```diff
@@ -919,6 +929,7 @@ CREATE TABLE IF NOT EXISTS `sc_data` (
   `val2` int(11) NOT NULL default '0',
   `val3` int(11) NOT NULL default '0',
   `val4` int(11) NOT NULL default '0',
+  PRIMARY KEY (`account_id`, `char_id`, `type`),
```

Comment:
char_id and type should be good enough.
No idea why we even have account id in this table.